radio/src/debug.h

```diff
@@ -348,8 +348,12 @@ class DebugTimer
   uint32_t _start_loprec;
 
   void evalStats() {
-    if (min > last) min = last;
-    if (max < last) max = last;
+    if (min > last) { 
```

Comment:
I don't know if it's more readable, but it doesn't respect our indentation (2 spaces)